Before: 18284f4f7f54e7be6b3f996e10eaf9a03bc0de2c41b410732ce21e9c895d5a42
After: f2920268a60bd90c5d24222d7ad0c443c65736254d43e7b87c0f89aa81f3cfb7
Patch: beea6f3a57cbe689e8cd3147694bb8d80a7ed357558e43d3896cdc7ec14adb24
--- lib/corvid/%name%_feature.rb.tt	2012-08-13 16:20:51.157407448 +1000
+++ /dev/null	2012-07-30 09:37:23.066384672 +1000
@@ -1,10 +0,0 @@
-require 'corvid/feature'
-
-class <%= class_name %> < Corvid::Feature
-
-  since_ver <%= since_ver %>
-
-  rake_tasks {
-  }
-
-end
--- resources/latest/corvid-features/%name%.rb.tt	2012-08-13 14:12:45.145097126 +1000
+++ /dev/null	2012-07-30 09:37:23.066384672 +1000
@@ -1,13 +0,0 @@
-install {
-  # empty_directory 'lib'
-  # copy_file       'hello.txt'
-  # copy_executable 'bin/hello'
-}
-
-# update {|ver|
-#   case ver
-#   when 2
-#     update stuff here
-#     empty_directory 'lib'
-#   end
-# }
